packages/openlayers/lib/map/create-map.ts

```diff
@@ -94,7 +109,12 @@ export async function createLayer(layerModel: MapContextLayer): Promise<Layer> {
       const olLayer = new VectorLayer({
         style: layerModel.style ?? defaultStyle,
       });
-      new WfsEndpoint(layerModel.url).isReady().then((endpoint) => {
+      const endpoint = (await getEndpoint(
```

Comment:
This is moving important logic in an unrelated file; instead we can define an error handler function like so:
`new WfsEndpoint(layerModel.url).isReady().catch(this.handleEndpointError).then((endpoint) => {`